packages/api/src/external/carequality/shared.ts

```diff
@@ -0,0 +1,10 @@
+import { PurposeOfUse } from "@metriport/shared";
+
+// TODO: adjust when adding purpose of use per cx
+export function createPurposeOfUse() {
```

Comment:
If possible, I'd add the `cxId` as parameter here, but NABD if you rather leave as is.

Reply:
hmm - I see the intention @leite08, however there is technically nothing stopping a cx for querying for different POUs when we support them

would suggest to leave as is, and make the TODO something like "adjust when we support multiple POUs"